spec/permissions.json

```diff
@@ -1,66 +1,80 @@
-
 {
   "groups": [
     {
       "name": "super user",
       "id": 0,
       "permissions": {
         "/**": {
-          "permissions": "All"
+          "permissions": 15
```

Comment:
What are all this `15`? It was nice before.

Reply:
It must be the default to serialize them as the numeric value, I can look into changing that.